Tasks/Npm/npminstall.ts

```diff
@@ -1,9 +1,13 @@
 import * as path from 'path';
-
 import * as tl from 'vsts-task-lib/task';
-
 import * as npmCustom from './npmcustom';
+import { NpmTaskInput} from './constants';
 
 export async function run(): Promise<void> {
-    return npmCustom.run('install');
+    let Command = 'install';
+    let verboseCheck = tl.getBoolInput(NpmTaskInput.Verbose, false);
+    if (verboseCheck) {
+        Command += " --verbose";
```

Comment:
Also, if this will be added to the custom task, I'd rather move this to `npmcustom.ts` to keep this file as simple as possible.